data-platform/autonomous-database/autonomous-json/python-duality-views/files/lab5_indexing.py

```diff
@@ -0,0 +1,56 @@
+import oracledb
+import json
+
+#connect with thick-client
+oracledb.init_oracle_client()
+connection = oracledb.connect(user="myapp",
```

Comment:
Add a note that YOUR_PASSWORD should be changed to a real password!
Better change it to a command line option. Hardcoded values are bad.